img-buildkite-64.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "testkitchen",
+    "image": "img-buildkite-64/testkitchen.png",
+    "category": "Buildkite",
+    "aliases": [
+      "test-kitchen",
+      "kitchen-ci"
+    ]
```

Comment:
I’d make the main “name” `test-kitchen`, rather than taking two very-similar names :)